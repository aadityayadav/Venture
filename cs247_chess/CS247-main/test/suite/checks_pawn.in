setup
+ K a1
+ P e4
+ k f5
done
- f5
+ k d5
done
- d5
+ k h8
done
