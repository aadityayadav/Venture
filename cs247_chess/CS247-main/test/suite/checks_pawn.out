The initial board cannot have a player in check!
The initial board cannot have a player in check!
_______k
________
________
________
____P___
________
________
K_______
